Examples/Tutorials/Tutorial-LiveTraffic/main.cpp

```diff
@@ -131,8 +130,8 @@ int main(int argc, char* argv[])
 		<< "   Default gateway:       " << dev->getDefaultGateway() << std::endl // get default gateway
 		<< "   Interface MTU:         " << dev->getMtu() << std::endl; // get interface MTU
 
-	if (dev->getDnsServers().size() > 0)
-		std::cout << "   DNS server:            " << dev->getDnsServers().at(0) << std::endl;
+	if (!dev->getDnsServers().empty())
+        std::cout << "   DNS server:            " << dev->getDnsServers().front() << std::endl;
```

Comment:
Please use 2 tabs instead of spaces

Reply:
Thanks, @seladb! I've updated the indentation to two tabs as suggested.